frontend/src/modules/entitieslist/components/Entity.css

```diff
@@ -95,6 +95,11 @@
     color: #FED271;
 }
 
+.entity.missing .status:before {
+    color: #5F7285;
+}
+
+
```

Comment:
We shouldn't need to add new rules. Instead, you can just modify the value of the `.entity .status:before {` rule.

Please also remove the very last rule at the bottom of the file. It's no longer needed, because the new "missing" color works well even on lighter background used when the entity is selected or hovered.
```css
.entity.missing.checked .status:before,
.entity.missing.selected .status:before,
.entity.missing:hover .status:before,
.entity.partial.checked .status:before,
.entity.partial.selected .status:before,
.entity.partial:hover .status:before {
    color: #3F4752;
}
```